charts/agent/values.yaml

```diff
@@ -173,6 +173,8 @@ ebpf:
   enabled: false
   # Define the kind of eBPF driver that will be used by the agent. Can be `legacy_ebpf` or `universal_ebpf`
   kind: legacy_ebpf
+# Run the Sysdig Agent container as privileged. When set to false, eBPF must be enabled
+privileged: true
```

Comment:
Let's add at least the values.schema.json to report that we have this field and must be a boolean.

Reply:
There are plenty of values not covered in the schema, any specific reason to add this one?

Reply:
We added it a few months ago (with the Universal eBPF) and we are adding new params, we don't go back and fill all the existing one ATM